atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts

```diff
@@ -0,0 +1,131 @@
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+
+import { Avatar, Button } from '@/shared/components'
+import { breakpoints, colors, spacing, typography } from '@/shared/theme'
+import { Link } from '@reach/router'
+
+export const Container = styled.section`
+  position: relative;
+  @media screen and (min-width: ${breakpoints.small}) {
+    margin-bottom: -75px;
+  }
+  @media screen and (min-width: ${breakpoints.medium}) {
```

Comment:
Not about code, but a big part of the image gets covered on bigger screens 
![image](https://user-images.githubusercontent.com/29527327/98790093-816f7800-2403-11eb-8a6c-125c011d506b.png)